composer.json

```diff
@@ -17,16 +17,16 @@
 	],
 	"require": {
 		"php": ">=5.3.2",
-		"symfony/form": "~2.3",
-		"symfony/http-kernel": "~2.3",
-		"symfony/translation": "~2.3",
-		"symfony/yaml": "~2.3"
+		"symfony/form": "^2.6.4|~3.0",
+		"symfony/http-kernel": "^2.6.4|~3.0",
+		"symfony/translation": "^2.6.4|~3.0",
+		"symfony/yaml": "^2.6.4|~3.0"
```

Comment:
well, this project would need to upgrade to Symfony 2.7 before upgrading to CraueFormFlowBundle. Is it an issue ? You will need to update the project anyway as 2.6 will be EOLed soon